plugins/modem-manager/fu-mm-device.c

```diff
@@ -1504,6 +1550,32 @@ fu_mm_device_attach(FuDevice *device, FuProgress *progress, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_mm_device_setup(FuDevice *device, GError **error)
+{
+	FuMmDevice *self = FU_MM_DEVICE(device);
+	g_autoptr(FuDeviceLocker) locker = NULL;
+
+	/* Create IO channel to send AT commands to the modem */
+	locker = fu_device_locker_new_full(device,
+					   (FuDeviceLockerFunc)fu_mm_device_io_open,
+					   (FuDeviceLockerFunc)fu_mm_device_io_close,
+					   error);
+	/*
+	 * firmware branch AT command may fail if not implemented,
+	 * clear error if not supported
+	 */
+	if (self->branch_at != NULL && !fu_mm_device_at_cmd(self, self->branch_at, TRUE, error))
+		g_clear_error(error);
```

Comment:
rather than doing this, please use `g_autoptr(GError) error_local = NULL` in this function scope. Then you can g_debug() the error_local->message on failure.